templates/gitlab-manager-runner.yaml

```diff
@@ -283,6 +283,7 @@ Resources:
                       MachineDriver = "amazonec2"
                       MachineName = "gitlab-docker-machine-%s"
                       MachineOptions = [
+                        "amazonec2-ami=ami-08c40ec9ead489470",
```

Comment:
The AMI is different for each region, right? I think we need to use a Mapping lookup base on AWS::Region

Reply:
I don't think I can put a lookup in the middle of the text block. How about this?